src/tsx/components/hero/ProjectHeroContent.tsx

```diff
@@ -0,0 +1,55 @@
+import { useState } from 'react';
+import ProcessCard from './../cards/ProcessCard';
+import { steps } from '../../../data/processSteps';
+import StepCircle from './../process/StepCircle';
+
+function ProjectHeroContent() {
+    const activeStep = steps.findIndex(step => step.isCurrent);
```

Comment:
You can use `useMemo` here to avoid recomputing the value on rerender and return the memorised value instead. It's not a big deal in this context, but it's definitely worth thinking about. 

```suggestion
    const activeStep = useMemo(() => steps.findIndex(step => step.isCurrent), []);
```